ui/app/src/components/CreateFolderDialog/CreateFolderContainer.tsx

```diff
@@ -242,14 +272,31 @@ export function CreateFolderContainer(props: CreateFolderContainerProps) {
 						}}
 						onChange={(event) => onInputChanges(applyFolderNameRules(event.target.value, { allowBraces }))}
 					/>
+					{rename && containsItemsInWorkflow && (
+						<Alert severity="warning" icon={false} sx={{ mb: 1 }}>
+							<FormControlLabel
+								disableTypography
+								control={<Checkbox onChange={onCancelPackagesAckChange} />}
+								label={
+									<Typography>
+										<FormattedMessage defaultMessage="The item is part of one or more publishing packages. Editing it will cancel the packages." />
```

Comment:
```suggestion
										<FormattedMessage defaultMessage="The folder contains items which take part in one or more publishing packages. Renaming it will cancel the packages." />
```